Server/ReasnAPI/ReasnAPI/Controllers/MeController.cs

```diff
@@ -47,15 +48,29 @@ public IActionResult GetCurrentUser()
         return Ok(updatedUser);
     }
 
+    [HttpGet]
+    [Route("image")]
+    [ProducesResponseType<ImageDto>(StatusCodes.Status200OK)]
+    public IActionResult GetCurrentUserImage()
+    {
+        var user = _userService.GetCurrentUser();
+        var image = _imageService.GetImagesByUserId(user.Id);
+
+        if (image is null)
+        {
+            return NotFound();
+        }
+
+        return File(image.First().ImageData, "image/jpeg");
+    }
```

Comment:
Why .First()? Shouldn't it return only 1 image for user?

Reply:
I changed ImageService - not it will return only one image.